core/schemas/entity.py

```diff
@@ -10,20 +9,12 @@
 from core.schemas.model import YetiTagModel
 
 
-class EntityType(str, Enum):
-    attack_pattern = "attack-pattern"
-    campaign = "campaign"
-    company = "company"
-    identity = "identity"
-    intrusion_set = "intrusion-set"
-    investigation = "investigation"
-    malware = "malware"
-    note = "note"
-    phone = "phone"
-    threat_actor = "threat-actor"
-    tool = "tool"
-    vulnerability = "vulnerability"
-    course_of_action = "course-of-action"
+# forward declarations
+class EntityType(str, Enum): ...
+
+
```

Comment:
Can we add a comment explaining how this is populated and pointing towards the code responsible for it? (Same for other such globals)

Reply:
fixed in d920b00f6eb932764c629a7c2fb3034aad0ffbe4